charts/logzio-k8s-events/templates/clusterrolebinding.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: {{ .Values.apiVersions.clusterRoleBinding }}
 kind: ClusterRoleBinding
 metadata:
   name: {{ include "logzio-k8s-events.fullname" . }}
+  namespace: {{ .Release.Namespace }} 
```

Comment:
I'm not sure if clusterrolebinding requires a namespace, please check.

Reply:
It doesn't, removed.